roofit/hs3/src/JSONFactories_HistFactory.cxx

```diff
@@ -592,50 +722,85 @@ class HistFactoryStreamer : public RooJSONFactoryWSTool::Exporter {
             int idx = 0;
             for (const auto &g : phf->paramList()) {
                ++idx;
-               RooPoisson *constraint = findPoissonClient(g);
-               if (constraint) {
-                  double erel = 1. / sqrt(constraint->getX().getVal());
-                  hist->SetBinError(idx, erel * hist->GetBinContent(idx));
-               } else {
-                  hist->SetBinError(idx, 0);
+               RooPoisson *constraint_p = findClient<RooPoisson>(g);
+               RooGaussian *constraint_g = findClient<RooGaussian>(g);
+               if (tot_yield.find(idx) == tot_yield.end()) {
+                  tot_yield[idx] = 0;
+                  tot_yield2[idx] = 0;
+               }
+               tot_yield[idx] += hist->GetBinContent(idx);
+               tot_yield2[idx] += (hist->GetBinContent(idx) * hist->GetBinContent(idx));
+               if (constraint_p) {
+                  double erel = 1. / sqrt(constraint_p->getX().getVal());
```

Comment:
```suggestion
                  double erel = 1. / std::sqrt(constraint_p->getX().getVal());
```
Please make the `std::` namespace explicit for math functions to make sure the right function is used